packages/fs-routes/index.ts

```diff
@@ -45,8 +45,8 @@ export default function fsRoutes(
   options: FsRoutesOptions = {}
 ): FsRoute[] {
   dir = path.resolve(process.cwd(), dir);
-  options.glob = options.glob || '**/*.js';
-  options.indexFileRegExp = options.indexFileRegExp || /(?:index)?\.js$/;
+  options.glob = options.glob || '**/*.+(js|cjs)';
```

Comment:
i don't feel comfortable baking in specific extensions like this.  you can pass in options.glob for your specific use case.

ES modules are still experimental in node, and the semantics around the `cjs`, and `mjs` file extensions are a bit confusing to me: https://redfin.engineering/node-modules-at-war-why-commonjs-and-es-modules-cant-get-along-9617135eeca1

i'd say leverage `options.glob` for now.

Reply:
Ah, if we can set the glob with the express-openapi package, that would be better than my solution indeed.